src/blueprint/blueprint.rs

```diff
@@ -224,8 +225,16 @@ impl Blueprint {
         let server = &blueprint.server;
         let mut schema = SchemaBuilder::from(&blueprint);
 
-        if server.enable_apollo_tracing {
-            schema = schema.extension(ApolloTracing);
+        #[cfg(all(feature = "apollo-tracing", not(target = "wasm32")))]
+        if let Some(ref apollo) = server.apollo {
+            let (graph_id, variant) = apollo.graph_ref.split_once('@').unwrap();
+            schema = schema.extension(ApolloTracing::new(
+                apollo.api_key.clone(),
+                apollo.platform.clone(),
+                graph_id.to_string(),
+                variant.to_string(),
+                apollo.version.clone(),
+            ));
```

Comment:
if we pass ApolloTracing as a schema modified, we can control when and how we pass it. This will allow us to pass it only from the server, without using a feature flag.